moto/glue/models.py

```diff
@@ -132,6 +135,11 @@ def create_database(
             database_name, database_input, catalog_id=self.account_id
         )
         self.databases[database_name] = database
+        if tags:
```

Comment:
It would be better to use `self.tag_resource(resource_arn, tags)` here.

No need for the if-statement, less code duplication, and it ensures that a call to `get_tags` always succeeds, regardless of whether the user supplied tags in the first place.

Keep in mind that you may need to change the signature of the `tag_resource` method, because the `tags`-argument then becomes `Optional`.

Reply:
Thanks for your quick feedback! Using `tag_resource` is a good idea, of course. I've implemented your suggestion.